sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -227,11 +227,19 @@ func runCtr(containerdSocket string, namespace string, containerID string, sourc
 					Destination: "/usr/local/bin/apiclient",
 					Source:      "/usr/bin/apiclient",
 				},
-				// Mount in the persistent storage location for this container
+				// Mount in the named persistent storage location for this container
 				{
 					Options:     []string{"rbind", "rw"},
 					Destination: "/.bottlerocket/host-containers/" + containerID,
 					Source:      "/local/host-containers/" + containerID,
+				},
+				// Mount in a generic persistent storage location for this container.
+				// This mount has the same source as the named persistent storage location above,
+				// but the destination is not tied to the containerID.
+				{
+					Options:     []string{"rbind", "rw"},
+					Destination: "/.bottlerocket/host-containers/current",
```

Comment:
What happens if the container is named "current"?  If the mount fails, I think we'll need to conditionally add this item to the list.

Reply:
I don't see the need for this part of the path `/.bottlerocket/host-containers`, why not 

```
Destination: "/host,
```

Reply:
The point of `/.bottlerocket` is that it's a namespace unlikely to be taken by any container image.  (`/host` could legitimately be in an image, and would be shadowed by our data.)  Then, `/host-containers` to segregate the different types of data we insert from the host, this being the persistent store per host container, as opposed to `/rootfs`, etc.

Reply:
@tjkirch I just launched an instance with a custom "current" control container. There were no errors to speak of and `/.bottlerocket/host-containers/` only contained one directory `current`. We can certainly add a check if you'd like, but nothing breaks as is.

@webern If we changed the location then we definitely wouldn't need to account for a container named "current" described above. I don't mind having the "current" location be somewhere else, but I think we need more consensus of where that somewhere else would be.

Reply:
> @tjkirch I just launched an instance with a custom "current" control container. There were no errors to speak of and `/.bottlerocket/host-containers/` only contained one directory `current`. We can certainly add a check if you'd like, but nothing breaks as is.

What mounts show up?  I just tested locally and found that it kinda-sorta recursively mounts.  I bind-mounted a at b, then a at b again, and see _three_ mounts, two for b and one for a.

It seems safest to me to conditionally include this extra entry based on name.

Reply:
There are a few people opposed to the variable approach because it makes it impossible to construct directories or even know paths at build time, in a Dockerfile, which seems like a fair argument.  I think we should fix this ^ with the conditional, and probably continue with this PR.